static/elements/chromedash-color-status.html

```diff
@@ -1,31 +1,43 @@
 <link rel="import" href="../bower_components/polymer/polymer.html">
 
-<polymer-element name="chromedash-color-status" attributes="value max"><!-- width height corner">-->
+<dom-module id="chromedash-color-status">
+  <link rel="import" type="css" href="../css/elements/chromedash-color-status.css">
```

Comment:
`type="css"` is polymer's hinting scheme: https://www.polymer-project.org/1.0/docs/devguide/styling.html#external-stylesheets. I really wish we'd change that :\